lia-ui/src/main/java/com/lithium/community/android/ui/components/fragments/LiCreateMessageFragment.java

```diff
@@ -159,6 +159,37 @@ public void onError(Exception exception) {
         }
     }
 
+    @Override
+    public void onActivityCreated(Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+        if(savedInstanceState != null){
+            outputFileUri = savedInstanceState.getParcelable(MediaStore.EXTRA_OUTPUT);
+            String message = savedInstanceState.getString(Intent.EXTRA_TEXT);
+            if(!TextUtils.isEmpty(message)) {
+                askQuestionBodyText = message.replaceAll("\\n", "<br />");
+            }
+            askQuestionSubjectText = savedInstanceState.getString(Intent.EXTRA_TITLE);
+            if(outputFileUri != null) {
+                handleImageSelection(null);
```

Comment:
why `null`? shouldn't it be `outputFileUri`?

Reply:
it's intent.